rest/models/request.go

```diff
@@ -48,3 +48,30 @@ func QueryParam(key, value string) RequestOption {
 		o.QueryParams.Add(key, value)
 	}
 }
+
+// Headers required to use the Launchpad product.
+const (
+	// HeaderEdgeID is a required Launchpad header. It identifies the Edge User requesting data.
+	HeaderEdgeID = "X-Polygon-Edge-ID"
+	// HeaderEdgeIPAddress is a required Launchpad header. It denotes the originating IP Address of the Edge User requesting data.
+	HeaderEdgeIPAddress = "X-Polygon-Edge-IP-Address"
+	// HeaderEdgeUserAgent is an optional Launchpad header. It denotes the originating UserAgent of the Edge User requesting data.
+	HeaderEdgeUserAgent = "X-Polygon-Edge-User-Agent"
+)
+
+// RequiredEdgeHeaders sets the required headers for the Launchpad product.
+func RequiredEdgeHeaders(edgeID, edgeIPAddress string) RequestOption {
+	return func(o *RequestOptions) {
+		if o.Headers == nil {
+			o.Headers = make(http.Header)
+		}
+
+		o.Headers.Add(HeaderEdgeID, edgeID)
+		o.Headers.Add(HeaderEdgeIPAddress, edgeIPAddress)
+	}
+}
+
+// EdgeUserAgent sets the Launchpad optional header denoting the Edge User's UserAgent.
+func EdgeUserAgent(userAgent string) RequestOption {
```

Comment:
I realized that technically it should be 
```suggestion
// EdgeUserUserAgent sets the Launchpad optional header denoting the Edge User's UserAgent.
func EdgeUserUserAgent(userAgent string) RequestOption {
```

Reply:
Totally agree, but is `EdgeUserAgent` still accurate. If you say yes then I'm convinced, just being a little pedantic.

Reply:
From a clarity standpoint, EdgeUserAgent is better imo. It also matches the naming of these other headers and vars in this PR (e.g. you went with HeaderEdgeID instead of HeaderEdgeUserID which I agree with)